docs/operate/reference/components/sensor/lorawan.md

```diff
@@ -0,0 +1,519 @@
+---
+title: "Configure a LoRaWAN network"
+linkTitle: "LoRaWAN"
+weight: 60
+type: "docs"
+description: "Configure a gateway and nodes to communicate over the LoRaWAN protocol."
+tags: ["sensor", "components", "lorawan", "gateway", "node"]
+icon: true
+images: ["/icons/components/sensor.svg"]
+---
+
+[LoRaWAN (Long Range Wide Area Network)](https://lora-alliance.org/) enables sensor communication spanning kilometers with minimal power usage.
+
+To collect LoRaWAN sensor data with Viam, use the [`lorawan`](https://app.viam.com/module/viam/lorawan) {{< glossary_tooltip term_id="module" text="module" >}}. You can find the module's source code in the [lorawan GitHub repository](https://github.com/viam-modules/lorawan).
+
+## Hardware requirements
+
+- a [supported LoRaWAN gateway](https://github.com/viam-modules/lorawan?tab=readme-ov-file#lorawan-gateway-models-provided)
+- a [supported LoRaWAN node](https://github.com/viam-modules/lorawan?tab=readme-ov-file#lorawan-sensor-models-provided)
+
+## Architecture
+
+You can use Viam to create LoRaWAN networks containing one gateway and multiple nodes.
```

Comment:
```suggestion
You can create LoRaWAN networks containing one gateway and multiple nodes.
```
Since we're already in the Viam docs we don't have to say this, but can just focus on their end goal